app/src/main/java/com/example/mvptask/view/adapter/ImageAndDateLoader.java

```diff
@@ -0,0 +1,20 @@
+package com.example.mvptask.view.adapter;
+
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.example.mvptask.R;
+import com.squareup.picasso.Picasso;
+
+public class ImageAndDateLoader {
```

Comment:
Image and date are two different things, each one should have their own separate helper class